Another_notebook.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import seaborn as sns; sns.set()
</pre><p><code>seaborn</code> should be i </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/agamemnonc/reviewnb-test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>